modules/serverless-configuring-burst-qps-for-net-kourier.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/kube-burst-qps-net-kourier.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-burst-qps-for-net-kourier_{context}"]
+= Configuring burst and QPS values for net-kourier
+
+The queries per second (QPS) value determines the number of client requests or API calls that are sent to the API server.
+
+The burst value determines how many requests from the client can be stored for processing. Requests exceeding this buffer will be dropped. This is helpful for controllers that are bursty and do not spread their requests uniformly in time.
+
+When the `net-kourier-controller` restarts, it parses all `ingress` resources deployed on the cluster, which leads to a significant number of API calls. Due to this, the `net-kourier-controller` can take a long time to start.
+
+You can adjust the QPS and burst values for the `net-kourier-controller` in the KnativeServing CR:
+
+.KnativeServing CR example
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  workloads:
+  - name: net-kourier-controller
+    env:
+    - container: controller
+      envVars:
+      - name: KUBE_API_BURST
+        value: "200" <1>
+      - name: KUBE_API_QPS
+        value: "200" <2>
+----
+<1> The QPS rate of communication between controller and the API Server. The default value is 200.
+<2> .. The burst capacity of communication between Kubelet and the API Server. The default value is 200.
```

Comment:
Should the `..` be here? It looks out of place.

Reply:
Thanks Jesse, it is fixed now.